src/app/components/related-posts.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Article from './article';
+
+export default function RelatedPosts({ slug }: { slug: string }) {
+    const relatedposts = [ 3, 4, 5 ];
+
+    return (
+        <div>
+            <span>Related Articles</span>
```

Comment:
Semantics please, this is clearly a heading. Spans and divisions should not be used everywhere like this.